### PR TITLE
User-story/2062-create-gui-element-to-define-multilinesections

### PR DESCRIPTION
**Pull request by martinbaerwolff**
OP#2062

### CODE REVIEW

File: OTAnalytics/plugin_ui/customtkinter_gui/line_section.py [hunk 1]
```diff
@@ -202,14 +205,22 @@ def update(self, coordinate: tuple[int, int], event_type: str) -> None:
...
+        if event_type == "left_mousebutton_up":
+            print("left_mousebutton_up")
             self.geometry_builder.add_coordinate(coordinate)
+        elif (
+            self.geometry_builder.number_of_coordinates() >= 1
+            and event_type == "mouse_motion"
+        ):
+            self.geometry_builder.add_temporary_coordinate(coordinate)
+        elif self.geometry_builder.number_of_coordinates() >= 2 and (
+            event_type in {"right_mousebutton_up", "return"}
```
Comment (briemla) [inline OTAnalytics/plugin_ui/customtkinter_gui/line_section.py:217]: Why a dictionary and not a list: `[]` or tuple: `()` ?

**Review by briemla — COMMENTED** [review]
Looks good so far. I could not reproduce the bug shown last time. I have only one question.

File: OTAnalytics/plugin_ui/customtkinter_gui/line_section.py (hunk 1 above)
Comment (martinbaerwolff) [reply OTAnalytics/plugin_ui/customtkinter_gui/line_section.py:217]: Given the definition of a set, it seems to be behave a dict without values :)

Comment (briemla) [reply OTAnalytics/plugin_ui/customtkinter_gui/line_section.py:217]: That's right, a set is in python like a dict without values. I have to lower by Java skills for this one ;-)

**Review by briemla — APPROVED** [review]

**Pull request merged**